Testscripts/Linux/build_test_vpp.sh

```diff
@@ -113,7 +107,7 @@ function build_test_vpp () {
 	sleep 10
 
 	# VPP Azure interfaces show as fortygigabit interfaces
-	vpp_hardware=$(ssh "${1}" vppctl show int | grep -iv 'local' | grep -i 'fortygigabit')
+	vpp_hardware=$(ssh "${1}" vppctl show int | grep -iv 'local' | grep -i 'GigabitEthernet')
```

Comment:
Comment?

Also you might check out with Azure Networking team if 40G or other speed is optional?

Reply:
I saw below actual output, and with the hardware changes, maybe we have 100G

```
VPP interfaces found: 
FiftyGigabitEthernet0             1     down         9000/0/0/0     
FiftyGigabitEthernet1             2     down         9000/0/0/0     

VPP interfaces found: 
FortyGigabitEthernet0             1     down         9000/0/0/0     
FortyGigabitEthernet1             2     down         9000/0/0/0
```

Reply:
Can we read from the system?

Reply:
See source code here https://docs.fd.io/vpp/17.10/d7/d1d/plugins_2dpdk_2device_2format_8c_source.html, there are many different types

Reply:
You can read the speed from ethtool. If you don’t want to do it now, open an issue and put it on my name. I can do it later